lib/sourceParams.js

```diff
@@ -56,6 +56,16 @@ module.exports = (params, app, appSchema) => {
         params.mode = 'development'
       }
 
+      // handle webpack=verbose, --wp, -p cli flags
+      if (flags.webpack === 'verbose' || flags.wp || flags.p) {
```

Comment:
this won't work as you expect. you need to check that each one of those flags === verbose, not merely check for verbose on one of the, then presence for the next two